zio-http/src/main/scala/zio/http/model/HttpError.scala

```diff
@@ -1,6 +1,6 @@
-package zio.http
+package zio.http.model
 
-import zio.http.HttpError.HTTPErrorWithCause
+import zio.http.model.HttpError.HTTPErrorWithCause
```

Comment:
In line 25, the `abstract class` is not sealed. Is this leaky or intended? I'd expect it is leaking. If so I could fix this here.

Reply:
Make it sealed